magni_demos/launch/dnn_rotate.launch

```diff
@@ -0,0 +1,28 @@
+<!--
+   Run the rotation  demo
+-->
+<launch>
+  <arg name="camera" default="raspicam_node"/>
+  <arg name="image" default="image"/>
+  <arg name="transport" default="compressed"/>
+
+  <!-- assume base nodes, motor, move_basic are already launched -->
+ 
+  <include file="$(find raspicam_node)/launch/camerav2_410x308_30fps.launch" />
+  
```

Comment:
Need to launch move basic.

(I would use the magni_nav launch file)

Reply:
OK, added